### ISSUE TITLE
maintained?

### ISSUE THREAD

**Issue by odyright** [KronicDeth/routing-securely-with-phoenix-framework#1]
Hello @all , is this repo still maintained / production ready?

**Comment by KronicDeth** [comment]
No, I haven't updated it since I last gave the presentation, but I'm not aware of anything that's changed in Phoenix to make the precautions unneccessary related the Phoenix.Channels and session kicking.

**Comment by odyright** [comment]
ok, I just started to learn dev with elixir...I like it but I'm confuse about the best practice for securing my apps! I'll try yours....

**Comment by KronicDeth** [comment]
Please open PRs with any changes you find, so we can say it's up-to-date after you're done.
